src/SFA.DAS.ApprenticeCommitments.Web.UnitTests/DateModelTests.cs

```diff
@@ -67,12 +67,11 @@ public void Valid_models_Date_property_contains_date_components(DateTime date)
         [TestCase(2020, 99, 01, "`99` is not a valid month")]
         [TestCase(0000, 01, 01, "`0000` is not a valid year")]
         [TestCase(2020, 02, 30, "`30` is not a valid day in `2020-02`")]
-
-        public void Invalid_models_Date_property_throws_exception(
+        public void Invalid_models_Date_property_returns_date_default(
```

Comment:
Do we still need `DateModel` in Apprentice Commitments Web?

Reply:
Good question I'm not sure